templates/faq.html

```diff
@@ -0,0 +1,18 @@
+{% extends 'base.html' %}
+{% load static %}
+{% block content %}
+
+<!-- help needed -->
+<!-- needs to be handled from backend urls and views -->
+<h1>FAQ</h1>
+
+<p> Lorem ipsum dolor sit amet consectetur, adipisicing elit. Beatae, error cumque. Esse, obcaecati necessitatibus quasi
```

Comment:
@JimLynx 
I think better here, would be to just leave out the Lorem text and paragraphs, and let's focus on getting that perhaps added as a new User Story #Mxx.